test/verify/check-reauthorize

```diff
@@ -93,11 +93,7 @@ class TestReauthorize(testlib.MachineCase):
         self.login_and_go("/playground/test")
         b.click(".super-channel button")
         b.wait_in_text(".super-channel span", 'result: ')
-
-        # TODO: this should only be 'access-denied'
-        legit_results = [f'result: {err}' for err in ('access-denied', 'authentication-failed', 'terminated')]
-        self.assertIn(b.text(".super-channel span"), legit_results)
```

Comment:
Seems [it's back](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19668-ba487574-20240325-115003-fedora-coreos-other/log.html#155) :cry: so perhaps drop that commit for the time being?

Reply:
I was kinda hoping to slay this one, to be honest...